src/lib/web-component/filter-timestamp/filter-timestamp-element.js

```diff
@@ -25,8 +25,11 @@ export class FilterTimestamp extends HTMLElement {
 
   setFilterRange = (info) => {
     const selectedEl = browser.menus.getTargetElement(info.targetElementId);
-    const selectedRow = selectedEl.closest('tr');
+    if (!selectedEl) {
+      return;
+    }
 
+    const selectedRow = selectedEl.closest('tr');
```

Comment:
Use the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) instead of adding an extra `if`.

Reply:
oh, yes. Added. Thanks :-)